Add Data Flow Diagram skeleton in the README

amanjeev opened the pull request:
There is a DFD in the README which is really helpful but services and
components are not detailed anywhere. This table needs to be populated
by the rest of the team who knows about the details of these components

Tickets:

- COGG-238

amanjeev committed:
Add Data Flow Diagram skeleton in the README

There is a DFD in the README which is really helpful but services and
components are not detailed anywhere. This table needs to be populated
by the rest of the team who knows about the details of these components

Tickets:

- COGG-238

shaikh-rashid commented on README.md:
```diff
@@ -147,3 +147,13 @@ To deploy CanDIGv2, follow one of the available install guides in `docs/`:
+The follwing table lists the details from the Data Flow Diagram in the "Overview" section.
+
+| Service/Component Name                  | Source            | Notes                  |
+| ----------------------                  |----------------   |--------                |
```
Looks good, could you start to add the additional components and tag who needs to provide the info for it?

amanjeev replied:
Yea I guess I would need to start somewhere. Perhaps @zxenia to begin the Katsu stuff. Thanks!

amanjeev committed:
Adds basic internal services and servers to table

Service and components tables needed to be populated for later
addition of details. This makes it easier for others who know more
about certain services to add details.

Tickets

- COGG-238

amanjeev commented:
If you can merge this skeleton then everyone can add stuff to this table as they work on this repo and see fit

amanjeev commented:
I had added the table to the README. Does this look good?

ljdursi reviewed (approved):
Looks good to me, but maybe call "CanDIG Server" something like "CanDIGv1 server (used for variant service)"?

amanjeev committed:
docs (README): name of candig_server
